src/java.management/share/classes/sun/management/spi/PlatformMBeanProvider.java

```diff
@@ -203,12 +203,8 @@ public default boolean shouldRegister() {
 
     /**
      * Instantiates a new PlatformMBeanProvider.
-     *
-     * @throws SecurityException if the subclass (and calling code) does not
-     *    have {@code RuntimePermission("sun.management.spi.PlatformMBeanProvider.subclass")}
      */
     protected PlatformMBeanProvider () {
-        this(checkSubclassPermission());
     }
 
     private PlatformMBeanProvider(Void unused) {
```

Comment:
This `PlatformMBeanProvider(Void)` constructor is no longer needed.   Same for `AgentProvider(Void)` constructor.

Reply:
Thanks, yes they can go...  Rebuilt and tested ok after the new commit.